core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/ControllerEvent.scala

```diff
@@ -18,11 +15,6 @@ object ControllerEvent {
       operatorStatistics: Map[String, OperatorRuntimeStats]
   ) extends ControlCommand[Unit]
 
-  case class BreakpointTriggered(
-      faultedTupleMap: mutable.HashMap[ActorVirtualIdentity, FaultedTuple],
-      operatorID: String = null
-  ) extends ControlCommand[Unit]
-
   case class ReportCurrentProcessingTuple(
       operatorID: String,
       tuple: Array[(ITuple, ActorVirtualIdentity)]
```

Comment:
is this still being used?

Reply:
no, but it is out of the scope of this PR.